app/views/shared/_footer.html.erb

```diff
@@ -1 +1,7 @@
-<footer class="l-footer"></footer>
+<footer class="l-footer">
+  <div class="l-constrained">
+    <div class="l-footer__inner">
+      <p class="l-footer__text"><%= t('footer.text') %> <%= link_to(t('footer.link_text'), "mailto:#{t('email_address')}") %></p>
```

Comment:
Instead of munging in the `mailto:` here you can use Rails' built in `mail_to` helper in place of `link_to`.

One thing to be aware of though is the ordering of the arguments. You'll need to switch the first and second arguments around as `mail_to` is intentionally stupid.